### PR TITLE
website is done

### PR DESCRIPTION
**Pull request by AlexeyBrambalov**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1 +1,146 @@
...
+    width: 15wv;
+}
+
+.contacts{
+    background-color: darkgrey;
+    width: 20wv;
+}
+
+.nav__list{
+    display: flex;
```
Comment (amcraethanan) [inline style.css]: well done on remembering to use flexbox from last week's lesson 👍

File: index.html [hunk 2]
```diff
@@ -7,10 +7,55 @@
...
+        </ul>
+    </nav>
+       
+    
+
+</header>
+
 <body>
-    <!-- Add your markup here -->
+    <div class="main__inner">
```
[the remark this answers is not in the document]
Comment (jamesgorrie) [reply index.html]: There's a [nice explanation here as to why this is important](https://www.w3.org/WAI/tutorials/page-structure/), not only for machines, but for humans too.

[The practical bits are explained here](https://www.w3.org/WAI/tutorials/page-structure/regions/).

File: style.css [hunk 3]
```diff
@@ -1 +1,146 @@
-/*  Add your styling here */
+.header{
+    width: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
```
Comment (jamesgorrie) [inline style.css]: These `0` values are not necessary as they are the default, they don't do any harm, but will make me wonder why you've used them if I look at them in the future...

File: style.css [hunk 4]
```diff
@@ -1 +1,146 @@
...
+}
+
+.logo{
+    background-color: grey;
+    width: 15wv;
+}
+
+.contacts{
+    background-color: darkgrey;
+    width: 20wv;
```
Comment (jamesgorrie) [inline style.css]: Do you mean `vw`? In which case this will be massive.

File: style.css [hunk 5]
```diff
@@ -1 +1,146 @@
...
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    justify-content: space-between;
+    align-items: stretch;
+    display: none;
+}
+
+.nav__item {
+    background-color: deeppink;
```
Comment (jamesgorrie) [inline style.css]: Great colour 🎨 !

File: style.css [hunk 6]
```diff
@@ -1 +1,146 @@
...
+    }
+    .main__info{
+        width: 25vw;
+        height: 25vh;
+    }
+    .main__image{
+        width: 75vw;
+    }
+
+}
```
Comment (jamesgorrie) [inline style.css]: I have found that keeping my styles together makes it easier to work out what's going on... e.g.

```CSS
.nav__list {
  /* Desktop */
}
@media (min-width: 768px) {
  .nav__list {
    /* mobile css */
  }
}

.main__info {
  /* Desktop */
}
@media (min-width: 768px) {
  .main__info {
    /* mobile css */
  }
}
```

This makes it easier to see what is different between the two versions as I generally work on 1 section of a site at once.

You would also be able to spot that you have written a lot of the same CSS for both versions!

**Pull request closed without merging**